### PR TITLE
Contribution / Components - ts templates update

### PR DESCRIPTION
**Pull request by itaytay**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: docs/contribution/CREATING_COMPONENTS.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Generator will:
 
 * create files according to structure defined in [Component Structure](./COMPONENT_STRUCTURE.md) document
 * add necessary `export`s to [index file](../../src/index.js), [testkits](../../testkit) and [stories index](../../stories/index.js)
+* add necessary mappings to [enzyme-typescript.template.ejs](../.wuf/testkits/enzyme-typescript.template.ejs), and [vanilla-typescript.template.ejs](../.wuf/testkits/vanilla-typescript.template.ejs)
```
Comment (simast) [inline docs/contribution/CREATING_COMPONENTS.md]: hmm, this is for `Generator will:` section - but our component generator does not do this change automatically. Could we move this somewhere else below and note that this is a manual process?

Comment (argshook) [reply docs/contribution/CREATING_COMPONENTS.md]: this sprint i will be working on automating those `.d.ts` files, so there wont be a need to adjust the template

i don't think we need to add this note, it's a temporary thing

Comment (itaytay) [reply docs/contribution/CREATING_COMPONENTS.md]: You're right. I misplaced it.
Updated and pushed

**Comment by itaytay** [comment]
@argshook @simast - I created a reminder for myself to check the status 2 weeks from now so we won't forget

**Pull request merged**